core/src/main/scala/io/qbeast/core/transform/Transformation.scala

```diff
@@ -47,7 +47,7 @@ trait OrdinalTransformation extends Transformation {
 /**
  * Identity transformation.
  */
-object IdentityTransformation extends Transformation {
+case class IdentityTransformation(newVal: Any) extends Transformation {
```

Comment:
Instead of `newVal,` would it be best if we call it `identityVal`?

Reply:
Since the class name relates to the IdentityTransformation and contains the "identity" string I think it is a bit redundant, if you think otherwise we can change it.

Reply:
It may be redundant, but newVal does not define what is this variable. We can name it `value` maybe, but it's not new.

Reply:
Okay, then let's be a bit redundant since it wont harm us anyway. I will rename it to identityVal